hooks/provider-pinned-versions/fixtures/pass_when_version_is_first.tf

```
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.0.0, < 2.0.0"
+
+  required_providers {
+    aws = {
+      version = "5.78.0"
```

Can we add a prerelease version here as a test?
```suggestion
      version = "5.78.0-pr-237.237.1.1"
```

---

I see it above... nice